app/controllers/v1/sessions_controller.rb

```diff
@@ -243,6 +237,23 @@ def callback_stats(status, saml_response = nil, failure_tag = nil)
       end
     end
 
+    # rubocop:disable Metrics/ParameterLists
+    def handle_callback_error(exc, status, response, level = :error, context = {},
+                              code = '007', tag = nil)
+      log_message_to_sentry(exc.message, level, extra_context: context)
+      redirect_to url_service(response&.in_response_to).login_redirect_url(auth: 'fail', code: code)
+      login_stats(:failure, response, exc) unless response.nil?
+      callback_stats(status, response, tag || code)
+      PersonalInformationLog.create(
+        error_class: exc,
```

Comment:
This attribute is called `error_class`, but it's just a String.  More importantly, it's how you will identify your `PersonalInformationLog` entries.  So, if you know what value will end up in the database, and you are sure it will be consistent, it's probably fine.  Otherwise, you might consider just changing this to a string literal so you can more easily/reliably retrieve your `PersonalInformationLog` entries.